app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\FormPostRequest;
+use App\Models\Post;
+use Illuminate\Http\Request;
+use Intervention\Image\ImageManagerStatic as Image;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+       $posts =  Post::all();
+        return view('home',['posts'=>$posts]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(FormPostRequest $request)
+    {
+
+        $attributes = $request->validated();
+
```

Comment:
```php
$attributes['user_id'] = $request->user()->id;
```

We can force this method / route to be only be accessible to authenticated users, thus preventing a user being able to create a post for another user.